code/lib/cli/src/sandbox-templates.ts

```diff
@@ -559,6 +560,28 @@ const benchTemplates = {
     },
     skipTasks: ['e2e-tests-dev', 'test-runner', 'test-runner-dev', 'e2e-tests', 'chromatic'],
   },
+  'bench/react-vite-default-ts/test-build': {
+    ...baseTemplates['react-vite/default-ts'],
+    name: 'Bench (react-vite/default-ts, test-build)',
+    inDevelopment: true,
+    isInternal: true,
+    modifications: {
+      skipTemplateStories: true,
+      testBuild: true,
+    },
+    skipTasks: ['e2e-tests-dev', 'test-runner', 'test-runner-dev', 'e2e-tests', 'chromatic'],
+  },
+  'bench/react-webpack-18-ts/test-build': {
```

Comment:
Does this naming scheme work? What about `bench-test/...` or `bench-test-build` instead?

Reply:
Also you should be able to spread the other bench tasks in if you want to keep it more DRY

Reply:
right